packages/documentation/scripts/generateAttribution.js

```diff
@@ -39,7 +39,7 @@ const gravatarURLForAuthor = (email) => {
       return "https://avatars.githubusercontent.com/u/49038?s=100&u=0b9ac5bf42a8ea2543a05191e150e0213456744e&v=4";
 
     default:
-      return crypto.createHash("md5").update(email).digest("hex");
```

Comment:
I'm not a 100% sure if anything else is needed to make them work. It seems like this should be enough according to the `createHash` function description in the `crypto` package.

Reply:
The question is whether or not this breaks anything, but maybe it doesn't; I can't imagine a md5 hash of an email address ever worked as a URL?

Reply:
Yeah, so this actually can be sha256:

- https://docs.gravatar.com/api/avatars/images/
- https://docs.gravatar.com/api/avatars/hash/

That should pass testing and also actually work.